docs/step-by-step/kubeapps-on-tkg/step-3.md

```diff
@@ -1,138 +1,115 @@
 
-## 3.X - RBAC in Application Repositories
+At the end of this step, the Kubeapps installation is configured with one or more application repositories. The next step is to [start using Kubeapps](./step-4.md).
 
-A Kubeapps `AppRepository` can be created by anyone with the required RBAC for that namespace.
+## Appendix: Role-Based Access Control (RBAC) in Application Repositories
 
-If you have cluster-wide RBAC for creating AppRepositories, you can still create an `AppRepository` whose charts will be available to users in all namespaces by selecting `All Namespaces` when creating the repository.
+A Kubeapps application repository can be created by anyone with the required RBAC privileges for that namespace.
 
-To grant a user (e.g., `USERNAME`) permissions to create `AppRepositories` objects in a specific namespace (e.g., `CUSTOM-NAMESPACE`), you must create `RoleBinding` objects associating `apprepositories-read` and `apprepositories-write` as follows.
+Users with cluster-wide RBAC privileges for creating application repositories  can still create an application repository whose charts will be available to users in all namespaces by selecting `All Namespaces` when creating the repository.
 
-> **TIP**: if you only require granting users the with permissions to deploy charts from `AppRepositories` in a specific namespace, create only the `apprepositories-read` `RoleBinding` object.
+To grant a user permission to create `AppRepository` objects in a specific namespace,  create `RoleBinding` objects associating `apprepositories-read` and `apprepositories-write` roles with that user, as shown below. Replace the `USERNAME` and `CUSTOM-NAMESPACE` placeholders with the corresponding username and namespace name.
+
+> **TIP**: If users only require permissions to deploy charts from application repositories in a specific namespace, it is sufficient to create only the `apprepositories-read` `RoleBinding` object.
 
 ```bash
-# Grant the user <USERNAME> read apprepositories permissions on the <CUSTOM-NAMESPACE> namespace
-kubectl -n <CUSTOM-NAMESPACE> create rolebinding <USERNAME>-apprepositories-read \
-    --user <USERNAME>
+# Grant the user USERNAME read apprepositories permissions on the CUSTOM-NAMESPACE namespace
+kubectl -n CUSTOM-NAMESPACE create rolebinding USERNAME-apprepositories-read \
+    --user USERNAME
     --clusterrole kubeapps:$KUBEAPPS_NAMESPACE:apprepositories-read
 
     --clusterrole kubeapps:$KUBEAPPS_NAMESPACE:apprepositories-write
 ```
 
 ```bash
-# Grant the user <USERNAME> write apprepositories permissions on the <CUSTOM-NAMESPACE> namespace
-kubectl -n <CUSTOM-NAMESPACE> create rolebinding <USERNAME>-apprepositories-write \
-    --user <USERNAME> \
+# Grant the user USERNAME write apprepositories permissions on the CUSTOM-NAMESPACE namespace
+kubectl -n CUSTOM-NAMESPACE create rolebinding USERNAME-apprepositories-write \
+    --user USERNAME \
     --clusterrole kubeapps:$KUBEAPPS_NAMESPACE:apprepositories-write
 ```
 
+TODO
```

Comment:
I'm not sure why this information is in this location. It does not seem related to RBAC.

Reply:
Yes, you're right, we can safely remove it (as it is already in the kubeapps docs). I left it there because it provides more context, but in this step-by-step guide, it's unnecessary perhaps.